src/components/Modals/Nfts/components/NftOverview.tsx

```diff
@@ -9,31 +10,26 @@ import { Row } from 'components/Row/Row'
 import { Text } from 'components/Text'
 import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
 import { markdownLinkToHTML } from 'lib/utils'
-import type {
-  SupportedZapperNetwork,
-  V2NftCollectionType,
-  V2ZapperNft,
-} from 'state/apis/zapper/validators'
-import { ZAPPER_NETWORKS_TO_CHAIN_ID_MAP } from 'state/apis/zapper/validators'
+import type { NftItem } from 'state/apis/nft/types'
+import type { V2NftCollectionType } from 'state/apis/zapper/validators'
 import { selectAssetById } from 'state/slices/assetsSlice/selectors'
 import { useAppSelector } from 'state/store'
 
 type NftOverviewProps = {
-  zapperNft: V2ZapperNft
-  zapperCollection?: V2NftCollectionType[]
+  nftItem: NftItem
+  nftCollection?: V2NftCollectionType[]
 }
 
-export const NftOverview: React.FC<NftOverviewProps> = ({ zapperCollection, zapperNft }) => {
+export const NftOverview: React.FC<NftOverviewProps> = ({ nftItem, nftCollection }) => {
   const translate = useTranslate()
 
-  const description = zapperCollection?.[0]?.collection.description
-  const collection = zapperNft?.collection
-  const tokenId = zapperNft?.tokenId
-  const address = collection?.address
-  const maybeNetwork = collection?.network
-  const nftStandard = collection?.nftStandard
-  const maybeChainId = ZAPPER_NETWORKS_TO_CHAIN_ID_MAP[maybeNetwork as SupportedZapperNetwork]
-  const maybeChainAdapter = getChainAdapterManager().get(maybeChainId as ChainId)
+  const description = nftCollection?.[0]?.collection.description
```

Comment:
Not in the `nftItem` yet since `nftCollection` comes from a separate query, consolidate me?

Reply:
Not for now - collection and NFT items (which do include *some* amount of collection data) are def. separate